[2025-05-28T07:25:39.235+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-28T07:25:39.379+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: image_processing_per_file_v2.preprocess_group.img_preprocess_task manual__2025-05-28T07:25:28.138938+00:00 map_index=5 [queued]>
[2025-05-28T07:25:39.473+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: image_processing_per_file_v2.preprocess_group.img_preprocess_task manual__2025-05-28T07:25:28.138938+00:00 map_index=5 [queued]>
[2025-05-28T07:25:39.480+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-05-28T07:25:39.563+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): preprocess_group.img_preprocess_task> on 2025-05-28 07:25:28.138938+00:00
[2025-05-28T07:25:39.656+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=43861) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-28T07:25:39.671+0000] {standard_task_runner.py:72} INFO - Started process 43920 to run task
[2025-05-28T07:25:39.650+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'image_processing_per_file_v2', 'preprocess_group.img_preprocess_task', 'manual__2025-05-28T07:25:28.138938+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmp3gk4j9h4', '--map-index', '5']
[2025-05-28T07:25:39.678+0000] {standard_task_runner.py:105} INFO - Job 48: Subtask preprocess_group.img_preprocess_task
[2025-05-28T07:25:39.953+0000] {task_command.py:467} INFO - Running <TaskInstance: image_processing_per_file_v2.preprocess_group.img_preprocess_task manual__2025-05-28T07:25:28.138938+00:00 map_index=5 [running]> on host 293edbf976f9
[2025-05-28T07:25:41.107+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='image_processing_per_file_v2' AIRFLOW_CTX_TASK_ID='preprocess_group.img_preprocess_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-28T07:25:28.138938+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-28T07:25:28.138938+00:00'
[2025-05-28T07:25:41.116+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-28T07:25:41.120+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-28T07:25:41.123+0000] {logging_mixin.py:190} INFO - Current task name:preprocess_group.img_preprocess_task state:running start_date:2025-05-28 07:25:39.389402+00:00
[2025-05-28T07:25:41.125+0000] {logging_mixin.py:190} INFO - Dag name:image_processing_per_file_v2 and current dag run status:running
[2025-05-28T07:25:41.126+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-28T07:25:49.065+0000] {python.py:240} INFO - Done. Returned value was: /opt/***/data/temp/1e690cae-6e85-4ea9-b0ac-c3fe12999187.png
[2025-05-28T07:25:49.184+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-28T07:25:49.193+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=image_processing_per_file_v2, task_id=preprocess_group.img_preprocess_task, run_id=manual__2025-05-28T07:25:28.138938+00:00, map_index=5, execution_date=20250528T072528, start_date=20250528T072539, end_date=20250528T072549
[2025-05-28T07:25:49.445+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-28T07:25:49.446+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-28T07:25:49.447+0000] {logging_mixin.py:190} INFO - Dag name:image_processing_per_file_v2 queued_at:2025-05-28 07:25:28.178051+00:00
[2025-05-28T07:25:49.448+0000] {logging_mixin.py:190} INFO - Task hostname:293edbf976f9 operator:_PythonDecoratedOperator
[2025-05-28T07:25:49.541+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-28T07:25:49.702+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-05-28T07:25:49.870+0000] {taskinstance.py:3901} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-05-28T07:25:49.895+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
